graphql/multiplication-connect/updateUserGamesPlayed.ts

```diff
@@ -0,0 +1,15 @@
+import { gql } from "@apollo/client";
```

Comment:
you could put this query in with both the games lost and games played queries so that you have a total of two queries and not three.  You would increment both the games played and games lost/won in the same query